app/assets/javascripts/map.js

```diff
@@ -28,17 +28,19 @@
         markers = L.layerGroup();
       }
       marker = null;
-      markerIcon = L.divIcon({
-        className: "map-marker",
-        iconSize: [30, 30],
-        iconAnchor: [15, 40],
-        html: '<div class="map-icon"></div>'
-      });
-      createMarker = function(latitude, longitude) {
+      markerIcon = function(alt_text) {
+        return L.divIcon({
+          className: "map-marker",
+          iconSize: [30, 30],
+          iconAnchor: [15, 40],
+          html: '<div class="map-icon" aria-label="' + alt_text + '"></div>'
```

Comment:
Here we concatenate the user provided title inside an `aria-label` attribute.
Should we take into account that users might include quotes or HTML that could break the attribute or the markup?